src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  gap: 1px 1px;
+  padding: 20px 10px 0;
+  flex-wrap: wrap;
+  max-width: 7 * $day-size + 5px + 20px;
+
+  @each $what-day, $i in $week {
+    &--start-day--#{$what-day}>:first-child {
+      margin-left: ($i - 1) * $day-size + ($i - 1);
+    }
+  }
+
+  @for $last-day from 28 to 31 {
+    &--month-lenght--#{$last-day}>:nth-child(n + #{$last-day + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    @include square ($day-size);
+    border: 1px solid #000;
+    box-sizing: border-box;
+    background-color: #eee;
+    @extend %text-style;
+    align-items: center;
+    justify-content: center;
+  }
+
+  @for $index from 1 through 31 {
+    &>:nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #0f6c8d;
+    transform: translateY(-20px);
+    transition: 0.5s;
+  }
```

Comment:
https://user-images.githubusercontent.com/94289038/191715995-021357d8-3121-44e5-b872-aae3297e463c.mov

lets make it smooth in both directions
pass transition from `hover` to element selector and add list of animated properties

Reply:
not fixed